app/guid-file/-components/file-detail-layout/component.ts

```diff
@@ -0,0 +1,19 @@
+// import { action } from '@ember/object';
+// import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+// import { tracked } from '@glimmer/tracking';
+// import { taskFor } from 'ember-concurrency-ts';
+// import Media from 'ember-responsive';
+
+interface Args {
+    rightColumnClosed: boolean;
+}
+
+export default class FileDetailLayout extends Component<Args> {
+    @action
+    noop() {
```

Comment:
Is this component file doing anything, or could this component be template-only?

Reply:
Just a boilerplate. I dunno whether there will be anything new added to the layout component so I left it there. Could be removed.

Reply:
Might as well take it out until we need it, I'm thinking.